static/app/views/issueList/actions/index.tsx

```diff
@@ -368,6 +369,16 @@ const SelectAllNotice = styled('div')`
   padding: ${space(0.5)} ${space(1.5)};
 `;
 
+const SelectAllNoticeBackground = styled('div')`
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: -1;
+  background: ${p => p.theme.background};
```

Comment:
🤔 What if we set two backgrounds, one the `theme.background` and the other a linear gradient with the yellow color.

That way we don't need an extra element?

Reply:
@evanpurkhiser Oh that's a great idea! Implemented as an optional `opaque` prop on <Alert /> in https://github.com/getsentry/sentry/pull/30821/commits/a68fdab55e0033a495f99974d59200a4fc4e597a.